### DISCUSSION TITLE
YouTube: Remove comment section

### DISCUSSION THREAD

**Discussion by alexandreteles** [ReVanced/revanced-discussions#418] [Question] [answered]
This feature is present in Vanced and removes the comment section, making the recommendations take the entire area below the description.

Additionally the patch could remove the comments button on shorts and stories as well.

**Comment by TheJeterLP** [comment, marked as answer]
Already implemented. Can be enabled in settings.

**Reply by oSumAtrIX** [reply]
There is no patch for it.

**Reply by TheJeterLP** [reply]
There is. Its included in the general ads patch. Was just disabled the whole time. Comments are litho components thats why theyre in there. Can be toggled using the settings.

**Reply by TheJeterLP** [reply]
Should be sepereated into its own patch tho. Same as most litho components

**Comment by ghost** [comment]
@alexandreteles Are you interested in all comment types removal (including shorts and live videos), or just the comments of common videos?

**Comment by alexandreteles** [comment]
@OxrxL I remove all access to shorts on my YouTube ReVanced builds, so generally speaking just the common videos would be fine but preferably all comments would be removed from the app, including shorts, stories and live videos.

Right now there is an option to remove comments from common videos so I guess this isn't that big of a deal, but having the option to remove them from shorts would be very nice to have.

**Reply by ghost** [reply]
Well...I have two mods to remove both shorts and live comments.

So...maybe, ReVanced team could port these to patcher.

**Reply by alexandreteles** [reply]
> Well...I have two mods to remove both shorts and live comments.
> 
> So...maybe, ReVanced team could port these to patcher.

Do you have a link for those patches?

**Reply by ghost** [reply]
> > Well...I have two mods to remove both shorts and live comments.
> > So...maybe, ReVanced team could port these to patcher.
> 
> Do you have a link for those patches?

Sure: https://filebin.net/d7hq542mbsu8uapq
